### DISCUSSION TITLE
emotion error

### DISCUSSION THREAD

**Discussion by MoZZANG** [vercel/next.js#49429] [Help]
### Summary
i installed emotion
![image](https://user-images.githubusercontent.com/98200337/236750909-b8c51c98-6ce9-42f7-b792-5232d545106a.png)

![image](https://user-images.githubusercontent.com/98200337/236750958-65f407dc-a511-4896-b64e-4006ca6c2bb5.png)

then i got error this
![image](https://user-images.githubusercontent.com/98200337/236751040-e79394e2-3641-47fd-aba6-e4bb2de88c85.png)

if i delete `emotion : true` then it's fine but for SSR,  i need that option. what should i do?

**Comment by icyJoseph** [comment]
I don't think emotion is fully supported yet: https://nextjs.org/docs/app/building-your-application/styling/css-in-js

Also, have you followed all of the CSS-in-JS steps? https://nextjs.org/docs/app/building-your-application/styling/css-in-js#configuring-css-in-js-in-app

**Reply by MoZZANG** [reply]
Thank you for replying. I see, emotion isn't ready yet for fully supporting

I'll try your recommend solution. Thanks again
